src/test/java/lv/ctco/javaschool/vote/boundary/VoteApiTest.java

```diff
@@ -57,149 +50,94 @@ void init() {
     }
 
     @Test
-    @DisplayName("Check data for user1,Mood happy = 1,without comments, Date is today")
-    void submitVote_Happy_without_Comment() {
+    @DisplayName("submitDailyVoteTest: Check data for user1,MoodStatus.EMPTY,without comments, Date is today")
+    void submitDailyVote_EMPTY_without_Comment() {
         when(userStore.getCurrentUser())
                 .thenReturn(user1);
-        FeedbackDto feedbackDto = new FeedbackDto();
-        feedbackDto.setMood(1);
-        feedbackDto.setComment("");
+        DailyVoteDto dailyVoteDto = new DailyVoteDto();
+        dailyVoteDto.setMood(MoodStatus.EMPTY);
+        dailyVoteDto.setComment("");
         doAnswer(invocation -> {
             DailyVote dailyVote = invocation.getArgument(0);
             assertEquals(user1, dailyVote.getUser());
-            assertEquals(LocalDate.now().toString(), dailyVote.getDate());
-            assertEquals(1, feedbackDto.getMood());
-            assertEquals("", feedbackDto.getComment());
+            assertEquals(LocalDate.now(), dailyVote.getDate());
+            assertEquals(MoodStatus.EMPTY, dailyVoteDto.getMood());
+            assertEquals("", dailyVoteDto.getComment());
```

Comment:
Please prefer `assertThat()` method, because it gives clearer error messages in case of test failure and is more flexible. More info here: https://confluence.ctco.lv/display/DEVCOM/Hamcrest+Matchers